corehq/apps/enterprise/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```

Comment:
i would also remove this file. not needed. when we need to use the `AppConfig` it's usually inserted in `__init__.py` and the name has `corehq.apps.` prepended to it